projects/demo/src/modules/app/stackblitz/project-files/stackblitz.less.md

```diff
@@ -1,7 +1,9 @@
+```less
 /**
 THIS IS A WORKAROUND FOR STACKBLITZ:
 DO NOT REPEAT, read https://taiga-ui.dev/getting-started#less
  */
 @import 'taiga-ui-global.less';
 @import 'theme/variables.less';
 @import 'theme/wrapper.less';
+```
```

Comment:
why did we change file format ?

Reply:
Because this file was needed only for automatic generation of stackblitz examples